File: Modul 1/PRAK110/PRAK110-2310817310011-MuhammadRizky.c
#include <stdio.h>
#include <math.h>

int main(){
    float alasSegitiga = 5, tinggiSegitiga = 12, sisiMiringSegitiga = sqrt((alasSegitiga * alasSegitiga) + (tinggiSegitiga * tinggiSegitiga));    
    printf("Diketahui :\nAlas = " "%.0f" " cm", alasSegitiga);
    printf("\nTinggi = " "%.0f" " cm", tinggiSegitiga);
    printf("\nJawab :");
    printf("\n\nSisi A = " "%.0f" " cm", alasSegitiga);
    printf("\nSisi B = " "%.0f" " cm", sisiMiringSegitiga);
    printf("\nSisi C = " "%.0f" " cm", tinggiSegitiga);
    printf("\nKeliling = " "%.0f" " cm", alasSegitiga + tinggiSegitiga + sisiMiringSegitiga);
    printf("\nLuas = " "%.0f" " cm", (alasSegitiga * tinggiSegitiga)/2);
    return 0;
}

File: Modul 1/PRAK110/PRAK110-2310817310011-MuhammadRizky.py
import math
alasSegitiga = 5; tinggiSegitiga = 12; sisiMiringSegitiga = int(math.sqrt(pow(alasSegitiga, 2) + pow(tinggiSegitiga, 2)))

print("Diketahui :\nAlas = ", alasSegitiga, "cm", "\nTinggi = ", tinggiSegitiga, "cm", "\n\nJawab :\nSisi A = ", tinggiSegitiga, "cm", "\nSisi B = ", sisiMiringSegitiga, "cm", "\nSisi C = ", alasSegitiga, "cm", "\nKeliling =", alasSegitiga + tinggiSegitiga + sisiMiringSegitiga, "cm", "\nLuas = ", int((alasSegitiga * tinggiSegitiga)/2), "cm")
